src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,47 @@
+import { useLocation } from 'react-router-dom';
+import { searchArticles } from '../../utils/NewsApi';
+
+const SearchForm = ({ setArticles, setSearchQuestion, setIsLoading }) => {
+  function handleSearch(e) {
+    e.preventDefault();
+    const token = localStorage.getItem('jwt');
+    const question = e.target.elements.search.value;
```

Comment:
`    const question = e.target.elements.search.value;
`

this is a pure **js** variant of inputs handling

To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components

you need to use `(`useState, value, onChange`)` here for the search form as well


and  you can use `useForm` also